src/StringLiteralGenerator/Utf8StringLiteralGenerator.cs

```diff
@@ -91,6 +92,7 @@ string generate(INamedTypeSymbol containingType, IEnumerable<(string methodName,
 
                 if (!string.IsNullOrEmpty(containingType.ContainingNamespace.Name))
                 {
+                    buffer.AppendLine("// <auto-generated />");
```

Comment:
This should be placed outside the `if` block.

Reply:
Sorry, I just rushed the patch. I'll fix it properly.